### DISCUSSION TITLE
Need help with zabbix lxc

### DISCUSSION THREAD

**Discussion by rofugos** [tteck/Proxmox#3458] [Help Needed] [answered]
The script works fine, but I cant connect to the db using the credentials. 
If anyone could write out the "configure DB connection"-page I would be very grateful

**Comment by Cancel357** [comment]
Same here

**Reply by juk1919** [reply]
same here

**Comment by tteck** [comment, marked as answer]
It's very simple

```
root@zabbix:~# cat zabbix.creds

zabbix Database Credentials

zabbix Database User: zabbix
zabbix Database Password: 4UJ3pZOXfodiG
zabbix Database Name: zabbixdb
root@zabbix:~# 
```
![Screenshot 2024-08-10 9 30 24 PM](https://github.com/user-attachments/assets/79ad180c-b440-44a3-a2b0-a04a55ddd4b6)

Maybe you're not entering the correct Database Name: `zabbixdb`

![Screenshot 2024-08-10 9 35 57 PM](https://github.com/user-attachments/assets/385bd907-7f5f-4e2f-b8cb-129f97950baf)

![Screenshot 2024-08-10 9 49 56 PM](https://github.com/user-attachments/assets/1f5d0073-117e-4fb0-97ae-7298c8dfa56f)

**Reply by Cancel357** [reply]
Thank you for this!! I got it working now.
